README.md

```diff
@@ -16,24 +16,37 @@ What's Cozy?
 [Cozy][cozy] is a platform that brings all your web services in the same private space.  With it, your webapps and your devices can share data easily, providing you with a new experience. You can install Cozy on your own hardware where no one's tracking you.
 
 
-What's API?
+What's cozy-client-js?
 ------------------
 
 `cozy-client-js` is a javascript library made by Cozy. It enables client-side  applications to make requests to the cozy stack.
 
 cozy-client-js is compatible with both cozy architectures.
 
-## This repository is a work-in-progress where the stack-v2 companion javascript library is being developed.
+## This repository is a work-in-progress where the cozy-stack companion javascript library is being developed.
 
-To learn more about the stack-v2, head over to [its repository](https://github.com/cozy/cozy-stack).
+To learn more about cozy-stack, head over to [its repository](https://github.com/cozy/cozy-stack).
 
 If you are getting started on cozy application development, you should follow this [tutorial](https://dev.cozy.io/clientsideapp.html) and use the current `cozy-browser-sdk`. Transitioning from the cozy-browser-sdk library to this one should imply minimal changes (see [transition doc](https://github.com/cozy/cozy-client-js/blob/master/docs/browser-sdk-transition.md) )
 
 
+Use
+---
+
+You can `import`/`require` cozy-client-js using webpack
+**TODO** test if `cozy-client-js` compatible with browserify, rollup, ect...
+
+You can also simply copy-paste the `dist/cozy-client.js` bundle file into your application, and include it in your application with  `<script src="./cozy-client.js">`.
+
+**polyfills** : cozy-client-js uses [fetch](https://fetch.spec.whatwg.org/) and bundle all necessary polyfills (promise, fetch, reGenerator). **TODO:** We should move the polyfills to a separate bundle and require them only if needed.
+
+Once you have the library included in your application, head over to [the doc](./docs/index.md) to see what you can do with it!
 
 Hack
 ----
 
+This section is only if you want to modify the cozy-client-js library. If you just want to make an app, head over to [the doc](./docs/index.md).
```

Comment:
Maybe we can move this Hack section to the contributing guide, to avoid confusion for simple consumer of this lib